src/app/static/src/app/store/downloadResults/mutations.ts

```diff
@@ -88,5 +100,11 @@ export const mutations: MutationTree<DownloadResultsState> = {
                 break
             }
         }
+    },
+
+    [DownloadResultsMutation.ResetIds](state: DownloadResultsState) {
+        state.summary.downloadId = "";
```

Comment:
For use after calibration, when any existing outputs necome invalid.

Reply:
Should we also stop polling if that's still going on?

Reply:
ah yes, good spot